### PR TITLE
Rename {enable|disable}_column_stats API

### PR DESCRIPTION
**Pull request by fabriziomello**
For better understanding we've decided to rename the public API from `{enable|disable}_column_stats` to `{enable|disable}_chunk_skipping`.

Disable-check: force-changelog-file
Disable-check: commit-count

### CODE REVIEW

File: tsl/test/expected/chunk_column_stats-14.out [hunk 1]
```diff
@@ -51,21 +51,21 @@ INSERT INTO sample_table VALUES (:'start_date'::timestamptz, 16, 14.6554, 47, 'n
 -- Non-int, date, timestamp cannot be specified as a min/max ranges for now
```
Comment (pallavisontakke) [inline tsl/test/expected/chunk_column_stats-14.out:51]: Does the test file name itself as `chunk_column_stats` seem alright? Would it need `chunk_skipping` in the name?

**Review by pallavisontakke — COMMENTED** [review]
Would we need changes to the file https://github.com/timescale/timescaledb/blob/main/.unreleased/pr_7020 ? It right now says:
`Implements: #7020: Add support for the chunk column statistics tracking.`

**Review by akuzm — APPROVED** [review]

File: tsl/test/expected/chunk_column_stats-14.out (hunk 1 above)
Comment (fabriziomello) [reply tsl/test/expected/chunk_column_stats-14.out:51]: To make it simple and fast I've decided to only rename the public API.

**Comment by fabriziomello** [comment]
> Would we need changes to the file https://github.com/timescale/timescaledb/blob/main/.unreleased/pr_7020 ? It right now says: `Implements: #7020: Add support for the chunk column statistics tracking.`

At this point would be better to change the release PRs.

**Review by gayyappan — APPROVED** [review]

**Pull request merged**